open-source-license-acknowledgements-and-third-party-copyrights.md

```diff
@@ -92,9 +92,9 @@ License: Creative Commons Attribution 3.0 BY
 https://github.com/erikdubbelboer/phpRedisAdmin/blob/master/README.markdown
 
 ### Selenium Docker
-Image: `Selenium/standalone-chrome:3.11`  
-License: Apache License 2.0  
-https://github.com/SeleniumHQ/docker-selenium/blob/master/LICENSE.md  
```

Comment:
two spaces are mandatory to preserve formatting 

now it looks like this
![Snímek obrazovky 2023-05-05 v 19 14 31](https://user-images.githubusercontent.com/1177414/236523382-c6495388-556f-43f6-8e1e-8e18c5038d6e.png)

Reply:
I forgot to mention to check only the last commit as it is only one different from the previous PR, but I have fixed it there so it is already fixed here